docs/recipes/sass-and-nunjucks-with-static-asset-revisioning.md

```diff
@@ -0,0 +1,79 @@
+# Sass and Nunjucks with static asset revisioning
+
+Static asset revisioning by appending content hash to filenames. Make sure to set the files to [never expire](http://developer.yahoo.com/performance/rules.html#expires) for this to have an effect.
+
+Sass and Nunjucks are here just to demonstrate how to implement static asset revisioning in a more real-world scenario. The content has will be appended in production, so it's recommended to add the following npm scripts to `package.json`:
+
+```json
+{
+  "scripts": {
+    "dev": "gulp-dev",
+    "build": "NODE_ENV=production gulp build"
+  }
+}
+```
+
+Now to install the required dependencies:
+
+```sh
+npm install --save-dev gulp gulp-rev gulp-rev-rewrite gulp-sass gulp-nunjucks gulp-if del
+```
+
+And, finally, `gulpfile.js`:
+
+```js
+const gulp = require('gulp')
+const sass = require('gulp-sass')
+const nunjucks = require('gulp-nunjucks')
+const rev = require('gulp-rev')
+const revRewrite = require('gulp-rev-rewrite')
+const gulpIf = require('gulp-if')
+const del = require('del')
+
+const isProd = process.env.NODE_ENV === 'production'
+
+function clean() {
+  return del('dist')
+}
+
+function styles() {
+  return gulp
+    .src('styles/style.scss')
+    .pipe(sass().on('error', sass.logError))
```

Comment:
I'm wondering if this pattern even makes sense anymore with `pipeline()`

Reply:
I would like to avoid this, too, but, as I said, I need help. 🙏